### DISCUSSION TITLE
Is there a limit to the number of files that can be set in Knowledge in the Community Edition?

### DISCUSSION THREAD

**Discussion by ymiyata910** [langgenius/dify#7008] [Help] [answered]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

We use the community version of Dify.
We know that you can set an Environment Variable for the number of files to upload and a size limit per file when creating Knowledge.

So, if we change the upper limit with an environment variable, can we set any number of documents to be used for Knowledge?
Also, is there any upper limit to the size of the uploaded files?

If there is an upper limit to the total number of files or file size that can be set per Knowledge, please let us know.

Thank you.

**Comment by crazywoola** [comment, marked as answer]
No, You can set any limit as you want.

**Reply by raywoocn** [reply]
How to set the upper limit of total number of files?
